app/package.json

```diff
@@ -43,7 +43,7 @@
     "mri": "^1.1.0",
     "p-limit": "^2.2.0",
     "primer-support": "^4.0.0",
-    "queue": "^4.4.2",
+    "queue": "^5.0.0",
```

Comment:
Based on [the diff](https://github.com/jessetane/queue/compare/4.4.2...5.0.1) between v4.4.2 and v5.0.0, the only change that could affect the logic of this module is https://github.com/jessetane/queue/pull/52 .

Based on that, my assessment is that this upgrade is very low risk.